x-pack/plugin/security/src/main/java/org/elasticsearch/xpack/security/support/SecuritySystemIndices.java

```diff
@@ -119,6 +128,7 @@ private SystemIndexDescriptor getSecurityMainIndexDescriptor() {
             .setSettings(getMainIndexSettings())
             .setAliasName(SECURITY_MAIN_ALIAS)
             .setIndexFormat(INTERNAL_MAIN_INDEX_FORMAT)
+            .setVersionMetaKey(SECURITY_VERSION_STRING)
             .setVersionMetaKey("security-version")
```

Comment:
The version meta key is set and immediately re-set here. Admittedly to the correct thing, but still.

Reply:
lol yes, setting it once should be enough :) Thanks!